project/LinkerdBuild.scala

```diff
@@ -258,7 +258,12 @@ object LinkerdBuild extends Base {
        |   -Dio.netty.allocator.numDirectArenas=${FINAGLE_WORKERS:-8}    \
        |   -Dcom.twitter.finagle.netty4.numWorkers=${FINAGLE_WORKERS:-8} \
        |   ${GC_LOG_OPTION:-}                                            \
-       |   ${LOCAL_JVM_OPTIONS:-}                                        "
+       |   ${LOCAL_JVM_OPTIONS:-}"
```

Comment:
Another question about the other flags related to CMS, should we still keep them around when we use JVM >9?

Reply:
I was just focused on getting rid of a warning. I think that may fall into a different scope of changes. I'm not sure right now what relies on what.